lxd/instance/drivers/driver_qemu.go

```diff
@@ -4738,6 +4738,11 @@ func (d *qemu) agentGetState() (*api.InstanceState, error) {
 	return status, nil
 }
 
+// isRunningStatusCode returns if instance is running from status code.
+func (d *qemu) isRunningStatusCode(statusCode api.StatusCode) bool {
```

Comment:
Isn't this identical to the lxc logic? If so, shouldn't it go to the common driver and avoid the duplication?

Reply:
Yeah good idea!